definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -141,5 +141,220 @@
+    "DisplayOrder": 6,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AosStarted",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "answerReceivedFromServiceAppRejected",
+    "Name": "Answer received",
+    "Description": "Answer received",
+    "DisplayOrder": 7,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "DefendedDivorce",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "coRespAnswerReceivedFromServiceAppRejected",
+    "Name": "Answer received",
+    "Description": "Answer received",
+    "DisplayOrder": 8,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "ServiceApplicationNotApproved",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "IssueAOSOffline_Respondent_FromServiceAppRejected",
+    "Name": "Issue AOS pack to respondent",
+    "Description": "Answer received",
+    "DisplayOrder": 9,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AosAwaiting",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "IssueAOSOffline_CoRespondent_FromServiceAppRejected",
+    "Name": "Issue AOS pack - Co-respondent",
+    "Description": "Answer received",
+    "DisplayOrder": 10,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "ServiceApplicationNotApproved",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "reissueFromServiceAppRejected",
+    "Name": "Reissue petition",
+    "Description": "Reissue petition",
+    "DisplayOrder": 11,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AwaitingReissue",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "transferCaseFromServiceAppRejected",
+    "Name": "Transfer from CTSC to RDC",
+    "Description": "Transfer from CTSC to RDC",
+    "DisplayOrder": 12,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "Issued",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "18/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "startAosFromServiceAppRejected",
+    "Name": "AOS started",
+    "Description": "AOS started",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "ServiceApplicationNotApproved",
+    "PostConditionState": "AosStarted",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/09/2020",
+    "CaseTypeID": "DIVORCE",
```

Comment:
I'm not sure hmm, did you do a merge with master or something like this? Seems you have each event twice :). Maybe a merging bug

Reply:
Yes, it was a merging bug.
I had rebased, it made sense. I tried to push, conflict, pull and then it essentially merged with master... undoing it all now

Reply:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.

Reply:
Ok, I have removed duplicates.

Please don't use rebase. Just merge code into your branch:

```
git pull origin master
```

to get the latest master. We squash everything before we merge it into master so it makes little sense to rebase your branch over and over again.

And as you can see you don't know what code ends up in the final version of your branch. Rebase is tricky. I dopn;t use it if I don't have to.